### PR TITLE
Fix element does not exist step defintion so it plays nicely with int…

### PR DESCRIPTION
**Pull request by mehalshah**
…ernet explorer.

### CODE REVIEW

File: dashboard/test/ui/step_definitions/steps.rb [hunk 1]
```diff
@@ -259,8 +259,7 @@ def replace_hostname(url)
 end
 
 Then /^element "([^"]*)" does not exist/ do |selector|
-  instances = @browser.execute_script("return $('#{selector}')");
-  instances.should eq []
+  @browser.execute_script("return $('#{selector}').length").should eq 0
```
Comment (islemaster) [inline dashboard/test/ui/step_definitions/steps.rb:262]: :+1: Seems safer.

**Comment by islemaster** [comment]
Have you run the updated features against Saucelabs?

File: dashboard/test/ui/step_definitions/steps.rb (hunk 1 above)
Comment (islemaster) [inline dashboard/test/ui/step_definitions/steps.rb:262]: Out of curiosity, what was IE returning for this step?  (Or for that matter, why were other browsers returning empty array?  Interesting cast.)

**Comment by mehalshah** [comment]
I tested against Saucelabs. I had one problem hitting localhost with the mobile browser but think it will be okay on test

**Comment by islemaster** [comment]
LGTM!

**Pull request merged**